Modelling.ipynb

```diff
@@ -0,0 +1,892 @@
+       "37971     35.0                57.3     38.0       30                55.9   \n",
+       "37972     35.0                66.7     49.0       28                63.6   \n",
+       "37973     30.0                68.4     53.0       22                70.7   \n",
+       "37974     48.0                40.7     22.0       31                41.5   \n",
+       "37975     37.0                48.6     23.0       28                45.1   \n",
+       "\n",
+       "       team_FOW  team_FOL  team_FO_win_percentage  team_oZS_percentage  \\\n",
+       "0            14        13                    51.9                 47.7   \n",
+       "1            18        33                    35.3                 37.5   \n",
+       "2            26        14                    65.0                 40.0   \n",
+       "3            21        17                    55.3                 53.5   \n",
+       "4            20        20                    50.0                 37.9   \n",
+       "...         ...       ...                     ...                  ...   \n",
+       "37971        21        25                    45.7                 46.6   \n",
+       "37972        24        18                    57.1                 61.3   \n",
+       "37973        24        21                    53.3                 70.4   \n",
+       "37974        20        22                    47.6                 42.3   \n",
+       "37975        26        27                    49.1                 39.3   \n",
+       "\n",
+       "       team_PDO           team    season  b2b  3_in_4  5_in_8  \n",
+       "0          93.1  Anaheim Ducks  20092010    0       0       0  \n",
+       "1         107.5  Anaheim Ducks  20092010    0       0       0  \n",
+       "2         110.3  Anaheim Ducks  20092010    0       0       0  \n",
+       "3         107.7  Anaheim Ducks  20092010    0       0       0  \n",
+       "4         100.0  Anaheim Ducks  20092010    1       1       0  \n",
+       "...         ...            ...       ...  ...     ...     ...  \n",
+       "37971      87.0  Winnipeg Jets  20222023    0       0       0  \n",
+       "37972     106.3  Winnipeg Jets  20222023    0       0       0  \n",
+       "37973     105.2  Winnipeg Jets  20222023    0       0       0  \n",
+       "37974     118.8  Winnipeg Jets  20222023    1       1       0  \n",
+       "37975      93.5  Winnipeg Jets  20222023    0       1       0  \n",
+       "\n",
+       "[37976 rows x 39 columns]"
+      ]
+     },
+     "execution_count": 147,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df['win_loss'] = df['win_loss'].map({'W': 1, 'L': 0})\n",
+    "df.fillna(df.mean(), inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "686e8002",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df2['winner_binary'] = (df2['winner'] == df2['homename']).astype(int)\n",
+    "df2['home_advantage'] = 1\n",
```

Comment:
Contains all the teams but the column is 1 if the home team won the match and 0 if not. Since the model was based on the probability of the home team winning the game

Reply:
My question is about `home_advantage`. What is the format of your dataset ? Is it a team view e.g. for the same game, there are 2 rows in your dataset (one per team) or is it a game view and you just added a variable for the home advantage ?

Reply:
It's a game view same as the main dataset and I just added a variable for the home_advantage

Reply:
Sounds good. Feel free to merge the PR then